backend/inspirehep/accounts/api.py

```diff
@@ -66,3 +80,46 @@ def get_current_user_orcid():
 
 def get_current_user_remote_orcid_account():
     return RemoteAccount.query.filter_by(user_id=current_user.get_id()).one_or_none()
+
+
+def get_allowed_roles_for_collections(collections, read_only=True):
+    private_collections = set(collections) - set(
+        current_app.config["NON_PRIVATE_LITERATURE_COLLECTIONS"]
+    )
+    roles = []
+    for col in private_collections:
+        roles.append(col + "-read-write")
```

Comment:
this will not work, for example `HEP HIDDEN-read-write` is not a valid role. We need to replace ` ` with `-` and maybe lowercase the collection